checker/src/main/java/org/checkerframework/checker/pico/jdk.astub

```diff
@@ -0,0 +1,355 @@
+import org.checkerframework.checker.pico.qual.Mutable;
+import org.checkerframework.checker.pico.qual.Immutable;
+import org.checkerframework.checker.pico.qual.ReceiverDependentMutable;
+import org.checkerframework.checker.pico.qual.Readonly;
+import org.checkerframework.checker.pico.qual.ObjectIdentityMethod;
+import java.util.Collection;
+
+package java.lang;
+
+@ReceiverDependentMutable
+class Iterable<E> {
+  Iterator<E> iterator(@Readonly Iterable this);
+}
+
+class AssertionError {
+  AssertionError(@Readonly Object var0);
+}
+
+class RuntimeException {
+  RuntimeException(String var0, Throwable var1);
```

Comment:
Should there be an annotation, e.g. on var1?

Reply:
Added readonly.